.codecov.yml

```diff
@@ -6,3 +6,13 @@ coverage:
       default:
         threshold: 0% # Ref: https://docs.codecov.io/docs/codecovyml-reference#coveragestatus
         target: auto
+
+# Since Backstage is a mono repo, flags here help in getting the code coverage of individual packages.
+# Documentation: https://docs.codecov.io/docs/flags
+flags:
+  packages-core:
```

Comment:
Would it make sense to name these just `core` and `core-api` here?

Reply:
I prefer `packages-core` to represent `packages/core` for avoiding namespace conflict. There can be a package X and plugin X at the same time. :)

Reply:
Plugins would be prefixed with `plugin-`

Reply:
Since we wouldn't have conflicts between packages and plugin names lets go with `core` and `core-api`?

Reply:
Aah, sounds good. Thank you.